src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx

```diff
@@ -43,10 +45,11 @@ const LiveRadioContainer = ({ idAttr, externalId, id }) => {
     liveRadio,
   );
 
-  const embedSource = embedUrl({
-    requestUrl: `${serviceId}/${id}/${lang}`,
+  const embedSource = getEmbedUrl({
+    mediaId: `${serviceId}/${id}/${lang}`,
     type: 'media',
     isAmp,
+    queryString: location.search,
```

Comment:
Would be nice if we could move this logic into `getEmbedUrl` rather than requiring each consumer to provide it, as it's not going to change from one page to the next - not important at this stage though